sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -658,9 +695,45 @@ public String getDefaultLanguage() {
      * @throws NullPointerException if {@code documents} is null.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<RecognizeLinkedEntitiesResultCollection>>
-        recognizeLinkedEntitiesBatchWithResponse(Iterable<TextDocumentInput> documents,
-            TextAnalyticsRequestOptions options) {
+    public Mono<Response<RecognizeLinkedEntitiesResultCollection>> recognizeLinkedEntitiesBatchWithResponse(
+        Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options) {
+        final RecognizeLinkedEntitiesOptions recognizeLinkedEntitiesOptions = new RecognizeLinkedEntitiesOptions();
+        String modelVersion = null;
+        boolean includeStatistics = false;
```

Comment:
Is the service expecting a null or a default?
Because [here](https://github.com/Azure/azure-sdk-for-java/pull/18984/files#diff-51db6a586d088a6fa478b349f6844aef3929b9655ea61f1e2a1f1858a5b200fdR203) we are sending null for incluseStatistics.

It should be consistent behavior?

Reply:
We allow user to set the value for primitive boolean. The default value for primitive boolean is always false. So it is consistent behavior comparing the GA version.

Reply:
ignored what I just said.

Reply:
Offline discussion conclusion. With the GA version, we can set the default value to false (which is the current behavior in service side if pass `null` as Boolean to service REST API). If in the future, service change the default value to true when null value is passed. We should change the default value to true manually (We already GAed with primitive boolean, calling isIncludeStatistics() will get false since the default value for primitive boolean is false),